connectors/sources/gcs.py

```diff
@@ -311,8 +313,12 @@ async def get_content(self, blob, timestamp=None, doit=None):
             )
             return
         logger.debug(f"Downloading {blob_name}")
-        content = b""
-        async with TemporaryFile("wb+") as async_buffer:
+        document = {
+            "_id": blob["id"],
+            "_timestamp": blob["_timestamp"],
+        }
+        source_file_name = ""
+        async with NamedTemporaryFile(mode="wb+", delete=False) as async_buffer:
```

Comment:
why the `+` ?

Reply:
I forgot to remove it. Thanks for reminding me.